applications/nrf5340_audio/src/bluetooth/le_audio_bis_headset.c

```diff
@@ -16,11 +16,13 @@
 #include "macros_common.h"
 #include "ctrl_events.h"
 #include "hw_codec.h"
+#include "channel_assignment.h"
 
 #include <logging/log.h>
 LOG_MODULE_REGISTER(bis_headset, CONFIG_LOG_BLE_LEVEL);
 
-BUILD_ASSERT(CONFIG_BT_AUDIO_BROADCAST_SNK_STREAM_COUNT <= 1,
+/* TODO: This really should be tested based on resources */
+BUILD_ASSERT(CONFIG_BT_AUDIO_BROADCAST_SNK_STREAM_COUNT <= 2,
```

Comment:
This has to be the number of BISes in the BIG or it will have no space for each BIS BASE. Once the BASE is scanned the required single stream can be decoded and rendered. 
I have left this as 2 and added a new config variable AUDIO_HEADSET_MAX_CHAN that is the maximum number of channels that can be decoded and rendered by the headset. In this case it is set to 1 and it now operates correctly.